demo/build.gradle.kts

```diff
@@ -49,6 +49,7 @@ kotlin {
                 implementation(compose.runtime)
                 implementation(cpnProject(project, ":material2"))
                 implementation(cpnProject(project, ":material3"))
+                implementation(cpnProject(project, ":lifecycle-viewmodel"))
```

Comment:
This could be removed since the "navigation" module depends on the "lifecycle-viewmodel" via `api` now.